docs/r2dbc.md

```diff
@@ -157,20 +157,23 @@ username.
 * Postgres: If the IAM email ends with `.gserviceaccount.com`, remove
   the `.gserviceaccount.com` suffix from the email.
 
-For example, if the full IAM user account
-is `my-sa@my-project.iam.gserviceaccount.com`,
-You would update the code, replacing these examples with the appropriate values
-for your database engine:
-
-|               | Mysql | Postgres             |
-|---------------|-------|----------------------|
-| <PROTOCOL>    | mysql | postgresql           |  
-| <IAM_DB_USER> | my-sa | my-sa@my-project.iam |
+For example, if the full IAM user account is 
+`my-sa@my-project.iam.gserviceaccount.com`, then the shortened database username
+would be `my-sa` for Mysql, and `my-sa@my-project.iam` for Postgres. 
```

Comment:
MySQL

Reply:
Ditto below. Postgres is common (over PostgreSQL), but I see most people write MySQL.

Reply:
I've updated both r2dbc.md and jdbc.md so that they consistently use "MySQL".